binr/r2pm/r2pm

```diff
@@ -145,10 +147,10 @@ commitSuicide() {
 		exit 1
 	fi
 	countDown "Self destroying in" 3
-	confirm "> Delete ~/.config.radare2" && (
-		rm -rf ~/.config/radare2
+	confirm "> Delete $RHOMEDIR" && (
+		rm -rf $RHOMEDIR
 	)
-	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ~/.config/radare2/prefix /"
+	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ${RHOMEDIR}/prefix /"
```

Comment:
uhm if this path contain spaces, we cant iterate over those paths like this :? and i dont want to use bashisms, so we may probably use : as token to split that list

Reply:
Not sure how to do that.